### ISSUE TITLE
Display bug whith css columns in Chrome

### ISSUE THREAD

**Issue by arnaudjuracek** [kingdido999/zooming#42]
When the image is inside a CSS column layout, the zoomed image stay in the column layout, and the bottom wrap on the next col when the zoomed image is grabbed (which is both weird and awesome).

- OS: OSX
- Browser and version: Chrome 60

### The grid
```scss
.grid {
  column-count: 2;
  column-gap: 1rem;

  figure {
    display: inline-block;
    margin: 0 0 0 0;
    width: 100%;
  }
}
```
![image](https://user-images.githubusercontent.com/2837959/30388778-09ff650e-98b1-11e7-95e1-137db6e0f428.png)

##### last image on the left col
![image](https://user-images.githubusercontent.com/2837959/30389317-c4c52026-98b2-11e7-961a-d335cf6e2110.png)

##### last image on the right col
![image](https://user-images.githubusercontent.com/2837959/30389332-d7254764-98b2-11e7-8d43-5756af7c4078.png)

**Comment by arnaudjuracek** [comment]
![image](https://user-images.githubusercontent.com/2837959/30389830-4b5b8052-98b4-11e7-9161-05495fd8e985.png)
The image wrapped in two cols ( ͡° ͜ʖ ͡°)

**Comment by kingdido999** [comment]
Hey @arnaudjuracek, thanks for the report! I will take a look this weekend.

**Comment by kingdido999** [comment]
@arnaudjuracek Would you mind creating a simple demo (on jsfiddle, codepen or anything like that) to reproduce this bug? You can add [zooming](https://cdnjs.cloudflare.com/ajax/libs/zooming/1.2.4/zooming.js) as an external resource.

**Comment by arnaudjuracek** [comment]
Here you go : [codepen.io/PJProM](https://codepen.io/anon/pen/PJProM)

**Comment by NewJenk** [comment]
Hi @kingdido999,

Have you had a chance to take a look at this issue?

Thanks

**Comment by kingdido999** [comment]
@NewJenk @arnaudjuracek I haven't done too much investigation, but looks like this issue happens on chrome only.

**Comment by kingdido999** [comment]
I don't really have a clue why it's happening in Chrome. I guess making an image wrapper with absolute position could fix this issue and other issues as well like #22 #34. But it would require quite some time for me to do the rewrite.

**Comment by andreaballerino** [comment]
Hi,
I've the same problem in Chrome and I fix this with:

img {
  transform: translateZ(0);
}

I hope help :)

**Comment by arnaudjuracek** [comment]
So you creating a new stacking context. Nice !
Thx, will try 👍

**Comment by kingdido999** [comment]
Fixed in the latest version `1.2.7`. Here's the [demo](https://codepen.io/anon/pen/RxjBxL).
